let(var a = 10, var b = a ^ 3)
{
    println(a+b);
    var x = a + b;
};


// Expected to fail as x is valid only inside 'let' context
print(x);
